config_defaults/subtests/docker_cli/systemd.ini

```diff
@@ -1,8 +1,30 @@
 [docker_cli/systemd]
-docker_timeout = 300
-#: Systemd unit file
+subsubtests = systemd_pull, systemd_build, systemd_run
+
+#: base image for Dockerfile. Not used in every test.
+dockerfile_base_image = centos:latest
+#: systemd unit file (basename only). Each subsubtest has a file by this
+#: name in our source dir; we copy it into /etc/systemd/system, replacing
+#: any '{foo}' strings in it with the corresponding 'foo' config setting.
```

Comment:
@edsantiago - thanks for cleaning it up. It is much cleaner & shorter than what I had.